### ISSUE TITLE
[Bug?] advanced wireless terminals not working on servers

### ISSUE THREAD

**Issue by Cominfiniti** [tom5454/Toms-Storage#229]
I have a lvl 4 beacon (which it does show on the terminal UI) yet I couldn't access it from far away or in nether, am I doing something wrong? here's the image
![image](https://user-images.githubusercontent.com/131906409/234662435-5ba90a35-65b5-4f24-a625-d3c4c1ace502.png)

And here's the image showing that it's not working
![image](https://user-images.githubusercontent.com/131906409/234662739-bc658e42-84fb-4f0c-a405-57a8df3608aa.png)

**Comment by tom5454** [comment]
Make sure the terminal is chunk loaded

**Comment by Cominfiniti** [comment]
> 

I see, okay let me try again, see if it works

**Comment by Cominfiniti** [comment]
the issue is solved, thanks a lot
